src/ios/CDVThemeableBrowser.m

```diff
@@ -487,6 +498,31 @@ - (BOOL)webView:(UIWebView*)theWebView shouldStartLoadWithRequest:(NSURLRequest*
             [self.commandDelegate sendPluginResult:pluginResult callbackId:scriptCallbackId];
             return NO;
         }
+    } else if ([self isSystemUrl:url]) {
+      // Do not iTunes store links from ThemeableBrowser as they do not work
+      // instead open them with App Store app or Safari
+      [[UIApplication sharedApplication] openURL:url];
+
+      // only in the case where a redirect link is opened in a freshly started
+      // ThemeableBrowser frame, trigger ThemeableBrowserRedirectExternalOnOpen
+      // event. This event can be handled in the app-side -- for instance, to
+      // close the ThemeableBrowser as the frame will contain a blank page
```

Comment:
For your test plan, have you tried open iTune link. Close ThemeableBrowser. Then open it again. Close ThemeableBrowser. Then open another link with a different URL pattern? Wouldn't `_framesOpened` persist across multiple instances of open and close?